src/modules/70-pipeline/components/PipelineStudio/PipelineStudio.module.scss

```
@@ -6,95 +6,8 @@
  */
 
 .container {
-  --pipeline-transparent: transparent;
-  --pipeline-grey-border: var(--grey-300);
-  --pipeline-light-grey: #e7e7e7;
-  --pipeline-blue-color: #25a6f7;
-  --pipeline-blue-bc: #021c3c;
-  --pipeline-tag-color: #4bace9;
-
-  width: 100%;
-  height: 100%;
-  overflow: hidden;
-
-  .content {
-    display: grid;
-    grid-template-rows: 42px 38px 1fr;
-    height: 100%;
-
-    .top-bar {
-      box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.13);
-      display: grid;
-      grid-template-columns: 1fr 125px 173px;
-      align-items: center;
-
-      .topButtons {
-        width: 56px;
-        border-radius: 14px;
-        background-color: var(--white);
-        display: inline-block;
-        font-size: 10px;
-        text-align: center;
-        color: #b1b1b1;
-        border: 1px solid var(--pipeline-grey-border);
-        margin-left: 12px;
-        padding-top: 2px;
-
-        &.selected {
-          color: var(--white);
-          cursor: not-allowed;
-          box-shadow: 0px 0px inset rgba(0, 0, 0, 0.5);
-          background-color: rgba(12, 48, 88, 0.4);
-        }
-      }
-
-      .savePublishBtn {
-        width: 153px;
-        height: 26px;
-        border: 1px solid #b8bfca !important;
-        color: var(--pipeline-blue-color) !important;
-        border-radius: 4px;
-      }
-    }
-
-    .secondary-bar {
-      border: 1px solid var(--pipeline-light-grey);
-      border-left-width: 0;
-      display: grid;
-      grid-template-columns: 70px 1fr 330px;
-      align-items: center;
-
-      .pipelineName {
-        display: inline-block;
-        padding-left: 20px;
-      }
-    }
-
-    .canvas {
-      height: 100%;
-    }
-  }
-
-  .left-bar {
-    background-color: var(--pipeline-blue-bc);
-    display: grid;
-    grid-template-rows: 1fr 50px;
-
-    .title {
-      writing-mode: tb-rl;
-      color: #c2c3c2;
-      font-size: 16px;
-      text-transform: uppercase;
-      padding: 0 16px;
-      transform: rotate(-180deg);
-    }
-
-    .logo-image {
-      padding: 16px 10px 10px;
-    }
-
-    .close-btn {
-      cursor: pointer;
-    }
-  }
+  --pipeline-approval-stage-color: var(--primary-7);
+  --pipeline-deploy-stage-color: #c023ff;
+  --pipeline-build-stage-color: #89cdff;
+  --pipeline-custom-stage-color: #f78f00;
```

Are there named colours we can use? If not, it might be a good idea to tell style lint to chill out on these
```suggestion
  /* stylelint-disable color-no-hex */
  --pipeline-deploy-stage-color: #c023ff;
  --pipeline-build-stage-color: #89cdff;
  --pipeline-custom-stage-color: #f78f00;
  /* stylelint-enable color-no-hex */
```